common/src/main/java/com/wynntils/mc/EventFactory.java

```diff
@@ -286,4 +289,14 @@ public static void onTickEnd() {
     }
 
     // endregion
+
+    // region Title Events
+    public static void onTitleSetText(Component component) {
+        post(new TitleSetTextEvent(component));
+    }
+
+    public static void onSubtitleSetText(Component component) {
+        post(new SubtitleSetTextEvent(component));
+    }
```

Comment:
All mixins should pass the packet to these methods in EventFactory, then get the component from the packet to create the event.